fast64_internal/f3d/f3d_material.py

```diff
@@ -570,8 +573,10 @@ def indentGroup(parent: UILayout, textOrProp: Union[str, "F3DMaterialProperty"],
 
         c = indentGroup(inputGroup, "Not useful:", True)
         c.prop(settings, "g_lod")
-        if isUcodeF3DEX1(bpy.context.scene.f3d_type):
+        if f3d.F3DEX_GBI:
```

Comment:
Note this also passes for f3dex2, whereas the previous `isUcodeF3DEX1()` check is specifically f3dex1

Reply:
Good catch! Sorry